portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,66 @@
+@import url(https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap);
+
+html,
+body {
+  height: 100%;
+  margin: 0 0;
+  background-color: #080808;
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  display: flex;
+  justify-content: center;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  height: 100%;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#header {
+  order: 0;
```

Comment:
if its already in the order you want, is the order necessary? Better to just rely on dom order.